experiments/functional/cstor-csi/csi-clone/clone.j2

```diff
@@ -0,0 +1,15 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: "{{ pvc_name }}"
+spec:
+  storageClassName: "{{ storage_class_name }}"
+  dataSource:
+    name: "{{ snapshot_name }}"
+    kind: VolumeSnapshot
+    apiGroup: snapshot.storage.k8s.io
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
```

Comment:
lets pass this as env. What if the size of parent volume is not 5Gi?

Reply:
resolved